be/src/http/default_path_handlers.h

```diff
@@ -20,6 +20,8 @@
 
 #include <stdio.h>
 
+#include "util/thread.h"
```

Comment:
You can move this include to cpp file to reduce includes in header files

Reply:
ok, done